src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -106,7 +106,7 @@ class HuiEntitiesCard extends LitElement implements LovelaceCard {
           ? html``
           : html`
               <div class="header">
-                <div class="name">${title}</div>
+                <div class="name">${this._hass.localize(title)}</div>
```

Comment:
We shouldn't lookup the title, it's set by humans. It's not supposed to be a lookup key.

Reply:
Alternative solution maybe regenerate lovelace config after language change?